CppCoreGuidelines.md

```diff
@@ -16556,12 +16556,12 @@ Template definition rule summary:
 
 * [T.60: Minimize a template's context dependencies](#Rt-depend)
 * [T.61: Do not over-parameterize members (SCARY)](#Rt-scary)
-* [T.62: Place non-dependent class template members in a non-templated base class](#Rt-nondependent)
+* [T.62: Place non-dependent class template members in a non-templated base class](#Rt-non-dependent)
```

Comment:
this changes an anchor, which may break external links (and is also a dictionary word)

Reply:
Thank you for the feedback!

I checked a bit more like below and personally think that "non-dependent" is a better option
Because "non-dependent" is being used in the standard
(+ this anchor is still working because I changed it in both sides)

#### nondependent
* In the stdandard (N4842)
  * 22times  = non-dependent 22
* In dictionaries
  * Google : X
  * Oxford : X
  * Collins : O
  * Dictionary.com : X
  * Merriam-Webster : O
  * Macmillan : X
  * Cambridge : X
  * Longman : X

Reply:
I resolve this conversation
If you have any suggestion please let me know !